cli-feed-v4.json

```diff
@@ -24443,6 +24453,379 @@
           "default": "false"
         }
       ]
+    },
```

Comment:
Please include a diff between this and the previous release for the v4-prerelease tag, as a gist linked from PR comments.

Reply:
https://gist.github.com/soninaren/84e4e139554c48e4cf38d93e12baaab3/revisions#diff-6eb5aa2af122281945d8d42cc285a04af319c927fec0b2e03ce47987c39985a2